app/components/assessment_report_component.html.erb

```diff
@@ -0,0 +1,57 @@
+<% if constraints.any? %>
+  <h3 class="govuk-heading-m"><%= t(".constraints_including_article") %></h3>
+  <ul class="govuk-list govuk-list--bullet">
+    <% constraints.each do |constraint| %>
+      <li><%= constraint.humanize %></li>
+    <% end %>
+  </ul>
+<% end %>
```

Comment:
I've moved this into a partial for my [PR](https://github.com/unboxed/bops/pull/782/files#diff-746b024295017db10b384f027a60eac862b6fae97a2837a496b7840ff9ba4556R1)

We'll be reusing this and the "planning history" sections for a few views now so probs best to use a shared source. Maybe it can live under a different directory though